node/Cargo.toml

```diff
@@ -0,0 +1,182 @@
+substrate-prometheus-endpoint = { workspace = true, default-features = true }
+
+# Cli specific
+frame-benchmarking-cli = { workspace = true, default-features = true, features = ["rocksdb"] }
+try-runtime-cli = { workspace = true, default-features = true, optional = true }
+
+# Cumulus
+cumulus-client-cli = { workspace = true, default-features = true }
+cumulus-client-consensus-aura = { workspace = true, default-features = true }
+cumulus-client-consensus-common = { workspace = true, default-features = true }
+cumulus-client-network = { workspace = true, default-features = true }
+cumulus-client-service = { workspace = true, default-features = true }
+cumulus-pallet-session-benchmarking = { workspace = true, default-features = true }
+cumulus-primitives-core = { workspace = true, default-features = true }
+cumulus-primitives-parachain-inherent = { workspace = true, default-features = true }
+cumulus-relay-chain-inprocess-interface = { workspace = true, default-features = true }
+cumulus-relay-chain-interface = { workspace = true, default-features = true }
+
+# Polkadot
+polkadot-cli = { workspace = true, default-features = true }
+polkadot-primitives = { workspace = true, default-features = true }
+polkadot-service = { workspace = true, default-features = true }
+xcm = { workspace = true, default-features = true }
+
+# Local
+altair-runtime = { workspace = true, default-features = true }
+centrifuge-runtime = { workspace = true, default-features = true }
+cfg-primitives = { workspace = true, default-features = true }
+cfg-types = { workspace = true, default-features = true }
+cfg-utils = { workspace = true, default-features = true }
+development-runtime = { workspace = true, default-features = true }
+pallet-anchors = { workspace = true, default-features = true }
+pallet-pool-system = { workspace = true, default-features = true }
+runtime-common = { workspace = true, default-features = true }
+
+# benchmarking
+frame-benchmarking = { workspace = true, default-features = true, optional = true }
+frame-system-benchmarking = { workspace = true, default-features = true, optional = true }
+
+# frontier
+fc-consensus = { workspace = true, default-features = true }
+fc-db = { workspace = true, default-features = true, features = ["rocksdb"] }
+fc-mapping-sync = { workspace = true, default-features = true }
+fc-rpc = { workspace = true, default-features = true }
+fc-rpc-core = { workspace = true, default-features = true }
+fp-consensus = { workspace = true, default-features = true }
+fp-evm = { workspace = true, default-features = true }
+fp-rpc = { workspace = true, default-features = true }
+fp-storage = { workspace = true, default-features = true }
+pallet-ethereum = { workspace = true, default-features = true }
+pallet-evm = { workspace = true, default-features = true }
+
+[build-dependencies]
+substrate-build-script-utils = { workspace = true, default-features = true }
+vergen = { workspace = true, default-features = true }
+
+[dev-dependencies]
```

Comment:
I'm not sure if we need any of these here.

Reply:
Good point! let me check it

Reply:
Shouldn't need those IMO

Reply:
Confirmed

Reply:
Confirmed